src/js/components/NameValuePair/NameValuePair.js

```diff
@@ -25,7 +25,12 @@ const NameValuePair = ({ children, name: nameProp }) => {
   let name;
   if (typeof nameProp === 'string' || typeof nameProp === 'number')
     name = (
-      <Text as="dt" textAlign={nameAlign} {...theme.nameValuePair.name}>
+      <Text
+        as="dt"
+        margin={column ? { bottom: theme.nameValuePair.margin } : 'undefined'}
```

Comment:
```suggestion
        margin={column ? theme.nameValuePair.margin : 'undefined'}
```
margin has margin.bottom, so we should just put it directly here